src/main/webapp/gameFunctions.js

```diff
@@ -0,0 +1,9 @@
+//This is the js file for gameBoard.html
+
+function getQuizStatus() {
+    fetch("/getQuizStatus").then(response => response.json()).then((tasks) => {
+        if(tasks != "NoNewButton") {
```

Comment:
It might be best to have this servlet return some actual json rather than just a hardcoded string. What if we change the string in the servlet to say "DontDisplayButton" for some reason? That would break this function. But if we have it return JSON, we can just check to see if the JSON is empty or not and go from there. So this if statement would be something like 

if (!tasks.isEmpty())

(I'm not sure of the exact syntax)